src/map/utils/battleutils.cpp

```diff
@@ -1905,7 +1905,35 @@ namespace battleutils
             damage = HandleStoneskin(PDefender, damage);
             HandleAfflatusMiseryDamage(PDefender, damage);
         }
-        damage = dsp_cap(damage, -99999, 99999);
+
+		if (PAttacker->objtype == TYPE_MOB && PDefender->objtype == TYPE_PC)	//Attacker is the Monster, damage gets decreased
```

Comment:
skip the TYPE_PC checks - there's no reason to restrict this bonus to players (for example, pets)